src/main/scala/com/amazon/deequ/analyzers/Distance.scala

```diff
@@ -15,11 +15,13 @@
  */
 
 package com.amazon.deequ.analyzers
+
 import org.apache.spark.mllib.linalg._
 import org.apache.spark.mllib.stat.Statistics
 import org.apache.spark.mllib.stat.test.ChiSqTestResult
-
+import scala.math._
 import scala.annotation.tailrec
+import com.amazon.deequ.metrics.{BucketValue}
```

Comment:
We can remove the`{}` around BucketValue

Reply:
corrected